mate-session/gsm-logout-dialog.c

```diff
@@ -20,6 +21,7 @@
  *
  * Authors:
  *	Vincent Untz <vuntz@gnome.org>
+ *      Gordon N. Squash
```

Comment:
Really?
I don't think that you are the author of this file.
One PR isn't enough to do that.
You're are not the maintainer of mate-session-manager!